forge/test/model_demos/high_prio/cnn/pytorch/test_detr.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+# Detr model having both object detection and segmentation model
+# https://huggingface.co/docs/transformers/en/model_doc/detr
+
+import pytest
+import forge
```

Comment:
General comment on import organization (sorted by hierarchy):
- Native python libraries (e.g. os, math, etc.)
- Third-party libraries (e.g. torch, numpy, etc.)
- Our libraries/files (e.g. forge)

Grouping them like this will make our python files a bit more easier to read and follow